exercises/make_it_modular/verify.js

```diff
@@ -197,4 +218,4 @@ function verify (callback) {
   verifyModuleUsed.call(this, callback)
 }
 
-module.exports = verify
\ No newline at end of file
+module.exports = verify
```

Comment:
I've made some changes to the indentation and added a line break in. Sorry this is my first pull request. I expect there will be warts in here.